packages/react-core/src/components/Alert/AlertActionLink.tsx

```diff
@@ -3,14 +3,14 @@ import { Button, ButtonVariant, ButtonProps } from '../Button';
 
 export interface AlertActionLinkProps extends ButtonProps {
   /** Content rendered inside the AlertLinkAction  */
-  children?: string;
+  children: string;
```

Comment:
Would this change be considered breaking? 

Everything else lgtm

Reply:
Yea I think I changed this because otherwise there's no way to guarantee an accessible name for the component and I had run into it during some of this work (this was part of why I had previously added the accessibility/breaking-change labels)

Probably safe to revert it (although not the best for a11y), checking now.

Reply:
We are starting breaking changes next milestone.  We can pull this in to the breaking change branch?

Reply:
Sure, I'll include this enhancement as part of https://github.com/patternfly/patternfly-react/pull/3822 and revert it from this PR so we can get the bug fixed.